scipy/stats/_stats_py.py

```diff
@@ -4057,7 +4062,7 @@ def alexandergovern(*samples, nan_policy='propagate'):
           (b**2*10 + 8*b*c**4 + 1000*b)))
 
     # (9) calculate statistic
-    A = np.sum(np.square(z))
+    A = np.sum(z**2, axis=0)
```

Comment:
why not `np.linalg.norm(z, axis=0)`? The array API equivalent was also recently added if I recall correctly.

Reply:
We could, but I think `vector_norm` would be taking the square root then squaring again.

Reply:
True, did not read properly.